napari/view_layers.py

```diff
@@ -139,39 +145,153 @@ def _make_viewer_then(add_method: str, args, kwargs) -> Viewer:
 
 @_merge_layer_viewer_sigs_docs
 def view_image(*args, **kwargs):
-    return _make_viewer_then('add_image', args, kwargs)
+    return _make_viewer_then('add_image', args, kwargs)[0]
 
 
 @_merge_layer_viewer_sigs_docs
 def view_labels(*args, **kwargs):
-    return _make_viewer_then('add_labels', args, kwargs)
+    return _make_viewer_then('add_labels', args, kwargs)[0]
 
 
 @_merge_layer_viewer_sigs_docs
 def view_points(*args, **kwargs):
-    return _make_viewer_then('add_points', args, kwargs)
+    return _make_viewer_then('add_points', args, kwargs)[0]
 
 
 @_merge_layer_viewer_sigs_docs
 def view_shapes(*args, **kwargs):
-    return _make_viewer_then('add_shapes', args, kwargs)
+    return _make_viewer_then('add_shapes', args, kwargs)[0]
 
 
 @_merge_layer_viewer_sigs_docs
 def view_surface(*args, **kwargs):
-    return _make_viewer_then('add_surface', args, kwargs)
+    return _make_viewer_then('add_surface', args, kwargs)[0]
 
 
 @_merge_layer_viewer_sigs_docs
 def view_tracks(*args, **kwargs):
-    return _make_viewer_then('add_tracks', args, kwargs)
+    return _make_viewer_then('add_tracks', args, kwargs)[0]
 
 
 @_merge_layer_viewer_sigs_docs
 def view_vectors(*args, **kwargs):
-    return _make_viewer_then('add_vectors', args, kwargs)
+    return _make_viewer_then('add_vectors', args, kwargs)[0]
 
 
 @_merge_layer_viewer_sigs_docs
 def view_path(*args, **kwargs):
-    return _make_viewer_then('open', args, kwargs)
+    return _make_viewer_then('open', args, kwargs)[0]
+
+
+def imshow(*args, viewer=None, **kwargs):
+    """Load data into an Image layer and return the Viewer and Layer.
+
+    Parameters
```

Comment:
`channel_axis` is in the tests, but isn't mentioned in the kwargs here.  Since this new function [has been proposed](https://github.com/napari/napari/pull/4858#issuecomment-1200505333) as the thing we'll use to deprecate the `channel_axis` argument in `add_image`, we definitely need that represented here.

Reply:
Isn't `channel_axis` only used by `add_image`? If so, I guess its fine to include if I mention that its for image only?

Reply:
`channel_axis` is not in the `ImageBase` signature. Its only added [here](https://github.com/napari/napari/blob/main/napari/components/viewer_model.py#L598). Why is that? What is the purpose of the method in the viewer_model?

Reply:
also, a couple of the default values in viewer_model are different from the ImageBase default values